java/BinExport/src/main/java/com/google/security/binexport/BinExport2Builder.java

```diff
@@ -311,8 +311,15 @@ private void buildFlowGraphs(Map<Long, Integer> basicBlockIndices)
               edge.setTargetBasicBlockIndex(targetId);
             }
             edges.add(edge.build());
+            // Possibly naive (and experimental) handling for switch tables
+          } else if (flow.isJump() && flow.isComputed()) {
```

Comment:
Regarding "possibly naive", I honestly have no clear idea. I found the Ghidra code hard to follow in this regard. There may be other situtations where `isComputed()` would return `true`, but I'm inclined to think that together with `isJump()` this may actually be the way to implement this.

We could also ask the Ghidra devs over on their project page, but for know I'd keep this as is.